components/common/logo/Logo.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+var curenetics_logo = require('./curenetics_logo_2.png');
+
+const Logo = styled.img`
```

Comment:
No need to complicate with absolute positioning. You should only have to set a max width on the logo, and the parent component style to:

`
display: flex;
align-items: center;
justify-content: space-between;
`